sample_files/problem_setting/examples/identical_checker.cpp

```diff
@@ -75,10 +75,28 @@ std::string readToken(char min_char = 0, char max_char = 127) {
   return token;
 }
 
+bool validateInt(const std::string &token) {
+  if (token == "0") {
+    return true;
+  }
+  auto i = 0u;
+  if (i < token.size() && token[i] == '-') {
+    ++i;
+  }
+  if (i >= token.size() || token[i] < '1' || '9' < token[i]) {
```

Comment:
Maybe you should explicitly note that `-0` is not a valid int, just in case some language considers it separate from `0`.

Reply:
This caveat is already in the documentation for `readInt`, should we add it here as a comment as well? I think the documentation is likely sufficient.